pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -471,3 +473,93 @@ def test_mi_falsey_name(self):
                                                             ('a', 'b')]))
         result = [x['name'] for x in build_table_schema(df)['fields']]
         assert result == ['level_0', 'level_1', 0, 1, 2, 3]
+
+
+class TestTableOrientReader(object):
+
+    @pytest.mark.parametrize("inp,exp", [
```

Comment:
these first 2 can just be functions, don't need to be in this class (should put next to the tests for the convert_pandas_type_to_json)